### ISSUE TITLE
Submitting Hard Corps to the list

### ISSUE THREAD

**Issue by revility** [dhewm/dhewm3-sdk#23]
Work has wrapped up on the updated version of hardqore2 for dhewm 3 hard corps.

I have git up with only the updated files.  Rest of the files are the same as what is here.  

https://github.com/revility/hardcorps

Outside of a few possible bug fixes in the future;  i would consider this pretty stable.  The documents folder has a full list of changes to the files & what for.  

Hard corps is already uploaded & will go live 10.15.21.

**Comment by DanielGibson** [comment]
Sorry for the late reply!

I created https://github.com/dhewm/dhewm3-sdk/tree/hardcorps based on your latest code.
I builds, but I haven't tested it yet.

I hope to update the dhewm3 mods page in the next days.

**Comment by DanielGibson** [comment]
updated https://dhewm3.org/mods.html as well

**Comment by revility** [comment]
Thanks for adding it to the list!  I know of a few bugs related to extremely high resolutions... but this related to the softcode/gui and not the hard code.  That I need to get a hotfix out for.  

Another bug is related to additional character select data, which I don't have enabled at all... thus isn't currently an issue.  Each playable character could have their own stats such as max health, jump height, speed, etc.

I hope people find this useful.  During development I found enough examples in D3 to create 2d gameplay ranging from Metroid to Mario with it... including head stomping lol.  I believe we covered all the basic needs for 2d AI, platforming, character movement, etc.
